R/MsBackend.R

```diff
@@ -125,6 +125,12 @@ NULL
 #'   spectrum. Returns an `integer` of length equal to the number of
 #'   spectra (with `NA_integer_` if not available).
 #'
+#' - `as.list` returns a `list` with the spectras' peak data. The length of
+#'   the list is equal to the number of spectra in `object`. Each element of
+#'   the list is a `matrix` with columns `mz` and `intensity`. For an empty
```

Comment:
Here you use:
> with columns `mz` and `intensity`

and for `replaceList` you use

> with columns `"mz"` and `"intensity"`

Please keep one of them. (I think the `"column"` version is easier to understand.)

Reply:
Yep. will change that. Thanks.